core-icon.html

```diff
@@ -167,7 +167,11 @@
         }
 
         this.iconset = this.computeIconset(this.icon);
-        this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        if (this.iconsetElement) {
+          this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        } else {
+          console.warn('iconset "' + this.iconset + '" not found!');
```

Comment:
It does seem like an error, but the exception is really unhelpful. How about throwing an exception with a more descriptive message?